framework/web/AssetBundle.php

```diff
@@ -147,7 +152,10 @@ public function registerAssetFiles($view)
             $view->registerJsFile($manager->getAssetUrl($this, $js), $this->jsOptions);
         }
         foreach ($this->css as $css) {
-            $view->registerCssFile($manager->getAssetUrl($this, $css), $this->cssOptions);
+            $css = ArrayHelper::toArray($css);
```

Comment:
Do we really need to use `ArrayHelper::toArray`? I think `$css = (array) $css` will be enough for this purpose.